extensions.json

```
@@ -1420,6 +1420,12 @@
       "id": "teabyii.ayu",
       "repository": "https://github.com/ayu-theme/vscode-ayu"
     },
+    {
+      "id": "tiehuis.zig",
+      "repository": "https://github.com/ziglang/vscode-zig",
+      "version": "0.2.5",
+      "checkout": "master"
```

This repository has no GitHub Releases or dedicated release tag, so there is no need to try and pin or checkout a specific version.

The best we can do here is publish straight from the default branch.

```suggestion
      "repository": "https://github.com/ziglang/vscode-zig"
```